tlv493d/tlv493d.go

```diff
@@ -407,9 +407,9 @@ func (d *Dev) ReadContinuous(frequency physic.Frequency, precision Precision) (<
 					// Try resetting the sensor to recover from errors
 					if err := d.initialize(true); err == nil {
 						if err := d.SetMode(newMode); err != nil {
-							log.Println("Unable to reset TLV493D mode:", err)
+							log.Printf("%s: unable to reset tlv493d mode: %v", d, err)
```

Comment:
One problem is that the String() doesn't include the port so it wouldn't help diagnosing as much as if it included the connection.

Reply:
Makes sense, I added the i2c address in the String(). However, it seems that there's no common pattern in all the logging and name methods, maybe a shared template could be enforced at least for the new drivers? Or maybe I just overlooked it :D

Reply:
No driver really log anything at the moment. It was intentional but I agree it can be annoying to diagnose issues. The system would benefit from being more opiniated here.